### ISSUE TITLE
Installation via composer not working

### ISSUE THREAD

**Issue by Brunty** [bolt/bolt#3415]
I'm installing bolt via composer:

    composer create-project bolt/composer-install mfyu-bolt --prefer-dist

    Do you want your web directory to be a separate folder to root? [y/n] y
    What do you want your public directory to be named? [default: public]
    Your project has been setup

The app/ folder is empty (no config, cache or database directories)

There's no .htaccess file in the public/ directory either.

Accessing the site gives the following: 

![Bolt errors](http://puu.sh/hkNa7/3dfa9ef4d1.png)

Those directories aren't writeable because they don't exist and I can't access mysite.dev/bolt because there's no .htaccess file to point bolt to index.php as the handler for URLs.

There's also an uncaught exception thrown:

```
Fatal error: Uncaught exception 'Bolt\Exception\LowlevelException' in /var/www/mfyu-bolt/vendor/bolt/bolt/src/Configuration/ComposerChecks.php on line 99
( ! ) Bolt\Exception\LowlevelException: in /var/www/mfyu-bolt/vendor/bolt/bolt/src/Configuration/ComposerChecks.php on line 99
Call Stack
#	Time	Memory	Function	Location
1	0.0010	243600	{main}( )	.../index.php:0
2	0.1472	1543136	Bolt\Configuration\ResourceManager->verify( )	.../index.php:8
3	0.1472	1543208	Bolt\Configuration\LowlevelChecks->doChecks( )	.../ResourceManager.php:405
4	0.1472	1543352	Bolt\Configuration\ComposerChecks->checkConfig( )	.../LowlevelChecks.php:74
5	0.1473	1543552	Bolt\Configuration\ComposerChecks->checkDir( )	.../ComposerChecks.php:43
````

**Comment by Brunty** [comment]
Just to add, this is using:

```
Installing bolt/composer-install (2.0.5)
  - Installing bolt/composer-install (2.0.5)
```

**Comment by GawainLynch** [comment]
@bobdenotter we should backport the fixes I made for this in #3150 & #3157

**Comment by bobdenotter** [comment]
Fixed by @GawainLynch in #3419
